python/templates/datamodel.h.jinja2

```diff
@@ -12,21 +12,27 @@
 namespace {{ package_name }} {
 
 using {{ package_name }}DataTypes = podio::utils::TypeList<
-{% for name in datatypes %}
-  {{ name }}{% if not loop.last %},{% endif %}
-{% endfor %}
+  {{ datatypes | join(', ')}}
+>;
+
+using {{ package_name }}DataCollectionTypes = podio::utils::TypeList<
+{%  if datatypes %}
+  {{ datatypes | join('Collection, ') ~ 'Collection' }}
```

Comment:
I didn't know this existed. This is pretty neat. I hope we still get what it does in a few months ;)

Reply:
Yeah I'm not sure it's much better than the for loop but apparently it's not so easy to concatenate to every element

Reply:
I think for this case it is OK as it is quite obvious what the result should be from the name and the surrounding context.